clients/deltalake/core/src/test/java/org/projectnessie/deltalake/ITDeltaLog.java

```diff
@@ -51,6 +57,91 @@ protected static void createDelta() {
         .set("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog");
   }
 
+  @Test
+  // Delta < 0.8 w/ Spark 2.x doesn't support multiple branches well (warnings when changing the configuration)
```

Comment:
for my benefit can you explain a bit more? If i recall correctly `0.6.0` is Spark2 only and `>0.7.0` is Spark3 only.

Reply:
that's delta/spark2 - it complains with a warning that the configuration has changed and "might not have been applied completely".